docs/content/dagster-cloud/non_isolated_runs.mdx

```diff
@@ -0,0 +1,40 @@
+# Choosing run isolation in Serverless
+
+Dagster Cloud Serverless offers 2 levels of run isolation: isolated and non-isolated.
+
+## Isolated runs
+
+Isolated runs take place in their own container with their own compute resources: 4 cpu cores and 16Gb of memory.
+
+These runs may take up to 3 minutes to start while these resources are provisioned.
+
+When launching runs manually, select `Isolate run environment` in the Launchpad to launch an isolated runs.
+Scheduled, sensor, and backfill runs are always isolated.
+
+## Non-isolated runs
+
+This can be enabled or disabled in deployment settings with
+
+```yaml
+non_isolated_runs
+  enabled: True
+```
+
+Non-isolated runs provide a faster start time by using a standing container for each code location.
+
+While launching runs from the Launchpad, leave `Isolate run environment` unchecked to launch a non-isolated run.
+Materializing assets from the UI also defaults to non-isolated.
+
+Non-isolated runs have access to fewer compute resources, and they are shared with other processes for a code location
```

Comment:
should get specific user code server resources

Reply:
yeah we should list the shared CPU/memory limits